src/components/AsyncSearchableSelect.tsx

```
@@ -85,73 +86,178 @@ const AsyncSearchableSelect = <
           },
         })}
         classNames={{
-          control: ({ isFocused }) => {
-            if (hasError && isFocused) {
-              return '!border-2 !border-danger-500';
+          ...( props?.classNames ? props.classNames : {} ),
+          control: (currentProps) => {
+            const userControl = props.classNames?.control;
+            // @ts-ignore
```

ESLint

[error] Use "@ts-expect-error" instead of "@ts-ignore", as "@ts-ignore" will do nothing if the following line is error-free.

(@typescript-eslint/ban-ts-comment)